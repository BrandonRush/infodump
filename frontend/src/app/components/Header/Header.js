import React, { Component } from 'react';
import { Container, Row } from 'react-bootstrap';

class Header extends Component {
  render() {
    return (
      <Container fluid className="p-5 mt-5 transparent nohighlight">
        <Row>
          <div className="col-sm text-center text-light" />
          <div className="col-sm text-center text-light h1 font-weight-bold">
            <div
              href="/#"
              role="link"
              className="noselect canselect text-light title"
              style={{ cursorDecoration: 'none' }}
            >
              <i className="icon ion-md-information-circle mr-2" />
              infodump
              {/* <span className="tiny-text theme-switch" role="img">
                ðŸŒ—
              </span> */}
            </div>
          </div>
          <div className="col-sm text-center text-light font-weight-normal" />
        </Row>
      </Container>
    );
  }
}

export default Header;
