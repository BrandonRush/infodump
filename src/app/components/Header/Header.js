import React from 'react';

const Header = () => {
  return (
    <header className="transparent nohighlight">
      <div role="link" className="noselect canselect title">
        <i className="icon ion-md-information-circle mr-2" />
        <span id="info">info</span>
        <span id="dump">dump</span>
        {false && <span id="beta">beta</span>}
        {/* <span className="tiny-text theme-switch" role="img">
                ğŸŒ—
              </span> */}
      </div>
    </header>
  );
};

export default Header;
