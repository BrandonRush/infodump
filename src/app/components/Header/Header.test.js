import React from 'react';
import Header from './Header';
import renderer from 'react-test-renderer';

test('Header renders without unexpected changes', () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
