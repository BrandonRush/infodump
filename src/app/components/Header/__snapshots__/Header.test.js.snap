// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders without unexpected changes 1`] = `
<header
  className="transparent nohighlight"
>
  <div
    className="noselect canselect title"
    role="link"
  >
    <i
      className="icon ion-md-information-circle mr-2"
    />
    <span
      id="info"
    >
      info
    </span>
    <span
      id="dump"
    >
      dump
    </span>
  </div>
</header>
`;
